src/lib/index.ts

```diff
@@ -3,4 +3,4 @@ export * from './components'
 
 // Styles
 import 'bootstrap/dist/css/bootstrap.min.css'
```

Comment:
Please check on export ambiguity in this file. Identity and streams are exporting the same member: `getSearchOptions` , `loading` and `onSearch`.